src/qutipinterface.jl

```diff
@@ -0,0 +1,119 @@
+@doc """
+Interface for `sesolve`[http://qutip.org/docs/3.1.0/apidoc/functions.html#module-qutip.sesolve] of QuTiP in QuDynamics
```

Comment:
I think this is misleading. The functions are only inspired the respective QuTiP interface. It might also be good to mention the differences to the latter.